scripts/plotting/global_latlon_map.py

```diff
@@ -336,6 +283,20 @@ def global_latlon_map(adfobj):
                             elif (redo_plot) and plot_name.is_file():
                                 plot_name.unlink()
 
+
+                            if weight_season:
+                                mseasons[s] = pf.seasonal_mean(mdata, season=s, is_climo=True)
+                                oseasons[s] = pf.seasonal_mean(mdata, season=s, is_climo=True)
```

Comment:
I think we'll want to use `odata` here:

```suggestion
                                oseasons[s] = pf.seasonal_mean(odata, season=s, is_climo=True)
```

Reply:
Well, that'd explain some of the strange behavior I saw in some recent testing.